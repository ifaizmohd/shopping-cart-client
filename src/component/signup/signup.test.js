import { shallow } from "enzyme";
import React from "react";
import Signup from "./signup.component";

describe("Signup component testing", () => {
  it("Shallow snapshot testing of signup component", () => {
    expect(shallow(<Signup />)).toMatchSnapshot();
  });
});
