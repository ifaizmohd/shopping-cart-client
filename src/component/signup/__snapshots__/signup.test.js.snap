// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup component testing Shallow snapshot testing of signup component 1`] = `ShallowWrapper {}`;
